src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/express/ExpressCaseServiceTest.java

```diff
@@ -0,0 +1,175 @@
+            FinremCaseData.builder()
+                .regionWrapper(RegionWrapper.builder()
+                    .allocatedRegionWrapper(AllocatedRegionWrapper.builder()
+                        .courtListWrapper(DefaultCourtListWrapper.builder()
+                            .nottinghamCourtList(BOSTON_COUNTY_COURT_AND_FAMILY_COURT)
+                            .build())
+                        .regionList(MIDLANDS)
+                        .midlandsFrcList(RegionMidlandsFrc.NOTTINGHAM)
+                        .build())
+                    .build())
+                .natureApplicationWrapper(NatureApplicationWrapper.builder()
+                    .natureOfApplicationChecklist(List.of(
+                        MAINTENANCE_PENDING_SUIT
+                    ))
+                    .build())
+                .fastTrackDecision(YesOrNo.NO)
+                .estimatedAssetsChecklistV2(UNDER_TWO_HUNDRED_AND_FIFTY_THOUSAND_POUNDS)
+                .build(),
+            // Is not under £250,000
+            FinremCaseData.builder()
+                .regionWrapper(RegionWrapper.builder()
+                    .allocatedRegionWrapper(AllocatedRegionWrapper.builder()
+                        .courtListWrapper(DefaultCourtListWrapper.builder()
+                            .nottinghamCourtList(NOTTINGHAM_COUNTY_COURT_AND_FAMILY_COURT)
+                            .build())
+                        .regionList(MIDLANDS)
+                        .midlandsFrcList(RegionMidlandsFrc.NOTTINGHAM)
+                        .build())
+                    .build())
+                .natureApplicationWrapper(NatureApplicationWrapper.builder()
+                    .natureOfApplicationChecklist(List.of(
+                        MAINTENANCE_PENDING_SUIT
+                    ))
+                    .build())
+                .fastTrackDecision(YesOrNo.NO)
+                .estimatedAssetsChecklistV2(UNABLE_TO_QUANTIFY)
+                .build(),
+            // Is a Fast Track Application
+            FinremCaseData.builder()
+                .regionWrapper(RegionWrapper.builder()
+                    .allocatedRegionWrapper(AllocatedRegionWrapper.builder()
+                        .courtListWrapper(DefaultCourtListWrapper.builder()
+                            .nottinghamCourtList(NOTTINGHAM_COUNTY_COURT_AND_FAMILY_COURT)
+                            .build())
+                        .regionList(MIDLANDS)
+                        .midlandsFrcList(RegionMidlandsFrc.NOTTINGHAM)
+                        .build())
+                    .build())
+                .natureApplicationWrapper(NatureApplicationWrapper.builder()
+                    .natureOfApplicationChecklist(List.of(
+                        MAINTENANCE_PENDING_SUIT
+                    ))
+                    .build())
+                .fastTrackDecision(YesOrNo.YES)
+                .estimatedAssetsChecklistV2(UNDER_TWO_HUNDRED_AND_FIFTY_THOUSAND_POUNDS)
+                .build()
+        );
+    }
+}
```

Comment:
Just a thought, could you write a happy path method that creates a happy path case?   Then you could the method to:

- get happy path case data for your first test.
- call the method in 'provideInvalidCaseData', but add a line after to disqualify the case.

Might read a bit nicer and curt down code.